tlv_lib/risc-v_shell_lib.tlv

```diff
@@ -77,7 +77,7 @@ m4+definitions(['
    |cpuviz
       @1
          /imem[m4_eval(M4_NUM_INSTRS-1):0]  // TODO: Cleanly report non-integer ranges.
-            $instr[31:0]         = /top|cpu<>0$instr;
+            $ANY = /top|cpu<>0$ANY;
```

Comment:
if pulls only $instr either way

Reply:
$instr is the memory contents. |cpu$instr is the one read from memory.

Reply:
I think it was still what you are saying. Reverted back to $instr.. in latest commit